src/java_bytecode/jar_file.h

```diff
@@ -15,27 +15,27 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <string>
 #include <vector>
 #include <map>
+#include <regex>
+#include <util/message.h>
 
-class jar_filet
+class jar_filet:public messaget
```

Comment:
Ok, but then you'll need to provide a `message_handlert` to configure things properly. Only that `message_handlert` coming in would have proper configuration in terms of verbosity and the way information should be printed. The code invoking `jar_filet` will have that, so it shouldn't be hard to do.